src/js/views/patients/schedule/layout.hbs

```diff
@@ -4,7 +4,9 @@
     <div class="schedule__list-search" data-search-region></div>
   </div>
   <div class="list-page__filters flex">
-    <div class="flex u-margin--r-24" data-select-all-region></div>
+    {{#unless isReduced}}
+      <div class="flex u-margin--r-24" data-select-all-region></div>
+    {{/unless}}
```

Comment:
Hide this region completely if it's the reduced schedule. 

To move the `All Filters` button to the left without there being an empty space because of this element's margin.

Reply:
But maybe I could remove `u-margin--r-24` from this `data-select-all-region` element. And add `u-margin--l-24` to the `data-filters-region` element instead... 🤔